### PR TITLE
Adding lgensinger/pkgparser

### PR DESCRIPTION
**Pull request by lgensinger**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by lgensinger** [comment]
@conda-forge/help-python, ready for review!

**Comment by lgensinger** [comment]
@conda-forge-admin, please ping team

File: recipes/pkgparser/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (ocefpaf) [inline recipes/pkgparser/LICENSE:21]: The license is in the package, so no need to put it here in the recipe.

File: recipes/pkgparser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 96c10814083212a0937b9d9359fb3b35905e64dc072f65eee53718efd2e9e2c0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (ocefpaf) [inline recipes/pkgparser/meta.yaml:14]: Looks like this is noarch and has two entrypoints.

```suggestion
  entry_points:
    - generate-package-docs=pkgparser.bin.generate_package_docs:main
    - parse-docs=pkgparser.bin.parse_docs:main
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  number: 0

```

File: recipes/pkgparser/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (ocefpaf) [inline recipes/pkgparser/meta.yaml:19]:
```suggestion
    - pip
```

File: recipes/pkgparser/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 96c10814083212a0937b9d9359fb3b35905e64dc072f65eee53718efd2e9e2c0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python
```
Comment (ocefpaf) [inline recipes/pkgparser/meta.yaml:18]:
```suggestion
    - python >=3.6
```

File: recipes/pkgparser/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    - setuptools
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/pkgparser/meta.yaml:24]:
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```

File: recipes/pkgparser/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - {{ name }}
```
Comment (ocefpaf) [inline recipes/pkgparser/meta.yaml:28]:
```suggestion
    - pkgparser
```

File: recipes/pkgparser/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
```
Comment (ocefpaf) [inline recipes/pkgparser/meta.yaml:29]:
```suggestion
  commands:
    - pip check
    - generate-package-docs --help
    - parse-docs --help
  requires:
    - pip

```

**Pull request merged**